frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.ts

```diff
@@ -59,13 +70,25 @@ export class UserDetailsComponent extends BaseComponent implements OnInit {
                     sex: resp.sex,
                     language: resp.language,
                     languageLevel: resp.languageLevel,
+                    birthDate: resp.birthDate,
                 });
+
+                this.userService.getTagNames().pipe(this.untilThis)
+                    .subscribe(tags => { this.selectedTags = tags; });
             });
+
+        this.tagService.getAllTags().pipe(this.untilThis).subscribe(
+            tags => { this.allTags = tags; },
+        );
```

Comment:
do not send two request, combine them

Reply:
will try to combine with `forkJoin`

Reply:
You should have one request, not a group of requests

Reply:
you puzzle me))

Reply:
I have an idea. Did you mean, that we always have to send to FE all the tags with corespondent user selection flag?
`TagDto{id: long, name: string, isSelected: bool}` This way will get all data in one request.